sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetSchemaSuite.scala

```diff
@@ -759,7 +759,7 @@ class ParquetSchemaSuite extends ParquetSchemaTest {
         nullable = true))),
     """message root {
       |  optional group f1 (MAP) {
-      |    repeated group map (MAP_KEY_VALUE) {
+      |    repeated group key_value (MAP_KEY_VALUE) {
```

Comment:
Would this be a possibly breaking change to files written as Parquet? may be a dumb question.

Reply:
1. `key_value` introduced by this fix: https://issues.apache.org/jira/browse/PARQUET-1879
2. I also use Parquet 1.11.1 to read this file: https://issues.apache.org/jira/browse/SPARK-32639
![image](https://user-images.githubusercontent.com/5399861/104973076-ab11d480-5a2e-11eb-9f1c-968ec63bce58.png)

Reply:
So, are you saying that there is no breaking change, @wangyum ?
@srowen 's question is asking the reason why we need this change, isn't it?

Reply:
Looking at the original PR, I think the change should be backward-compatible (`map` annotation can still be handled  on the read path).

Reply:
Seems better to test both.